systemtest/src/test/java/io/strimzi/systemtest/migration/MigrationST.java

```diff
@@ -328,7 +329,7 @@ private void setupMigrationTestCase(TestStorage testStorage, boolean zkDeleteCla
             .withBootstrapAddress(KafkaResources.plainBootstrapAddress(testStorage.getClusterName()))
             .withTopicName(continuousTopicName)
             .withMessageCount(continuousMessageCount)
-            .withDelayMs(1000)
+            .withDelayMs(500)
```

Comment:
what does exactly this set? Delay between messages? Sorry but it's not clear to me even from the PR description.

Reply:
Yes, delay between messages.

Reply:
So how does sending messages faster help here?

Reply:
It doesn't help in any way. I just wanted to speed up the test and not wait additional 3 minutes or so before the messages will be send/received.

Reply:
And it's mentioned in here `long wait before the continuous clients are finished - changed the delay to 500 ms (from 1000ms)`

Reply:
> And it's mentioned in here long wait before the continuous clients are finished - changed the delay to 500 ms (from 1000ms)

Which was exactly the sentence which was not clear to me :-D